tests/model_registry/test_model_registry_creation.py

```diff
@@ -1,81 +1,107 @@
 import pytest
+import re
 from typing import Self
 from simple_logger.logger import get_logger
 
 from ocp_resources.data_science_cluster import DataScienceCluster
-from utilities.constants import Protocols, DscComponents, ModelFormat
-from model_registry import ModelRegistry
-
+from ocp_resources.pod import Pod
+from ocp_resources.namespace import Namespace
+from utilities.constants import DscComponents
+from tests.model_registry.constants import MR_NAMESPACE, MR_OPERATOR_NAME, MODEL_NAME, MODEL_DICT, CUSTOM_NAMESPACE
+from pytest_testconfig import config as py_config
+from model_registry import ModelRegistry as ModelRegistryClient
+from model_registry.types import RegisteredModel
+from kubernetes.dynamic import DynamicClient
 
 LOGGER = get_logger(name=__name__)
-MODEL_NAME: str = "my-model"
 
 
 @pytest.mark.parametrize(
-    "updated_dsc_component_state",
+    "model_registry_namespace, updated_dsc_component_state_scope_class",
     [
         pytest.param(
+            {"namespace_name": CUSTOM_NAMESPACE},
```

Comment:
for easier debugging, please set the namespace to match the test and not set the same ns name for in the 2 modules

Reply:
Sure.